docs/email_reviewing.md

```diff
@@ -0,0 +1,29 @@
+# Email Reviewing
+
+This is where we need the most help. When emails are first written, they need to be fact-checked and lightly edited for typos. First of all, you will need to make a GitHub Account. Then you will need some permissions to edit the labels on issues. Submit an issue to request that [here](https://github.com/defund12/defund12.org/issues/new?assignees=&labels=admin-attention&template=TRIAGE_ACCESS.md&title=Triage+access+for+%5BUSERNAME%5D).
+
+You can talk to other members of the team on the corresponding channel on [our Discord server](https://discord.gg/9zghS4V).
+
+## Find emails that review
+
+Those will be in [issues](https://github.com/defund12/defund12.org/issues?q=is%3Aissue+is%3Aopen+label%3Anew-request) that have the `new-request` label.
+
+## Remove spam/duds
```

Comment:
This responsibility was absorbed from the triage team, I thought that the triage team seemed like it's responsibilities were a bit tiny and silly, so may as well simplify and remove the number of steps in the pipeline.

Reply:
Triage is still important. While the actions for triage are small, the impact is great, freeing up the reviewers to spend their time actually instead of constantly checking for new issues and [replying, labeling, and closing](https://docs.google.com/document/d/1iXNxrfU5aODm0_Wd7zQClNbpnBrmKfIYsrE4g0773g8) properly. Until we have a bot smart enough to do this successfully, I advise keeping triage as a separate team function.